docs/docs/reference/api-reference/lifecycle/v1alpha2/index.md

```diff
@@ -10,7 +10,7 @@ Reference information for lifecycle.keptn.sh/v1alpha2
 
 ## lifecycle.keptn.sh/v1alpha2
 
-Package v1alpha2 contains API Schema definitions for the lifecycle v1alpha2 API group
```

Comment:
how did this line get removed with the change in this PR?

Reply:
no idea, package as autogenerated 🤷‍♀️

Reply:
yes and that part was autogenerated before as well 😅